metricflow/model/parsing/dir_to_model.py

```diff
@@ -237,19 +236,20 @@ def parse_config_yaml(
 
 def parse(  # type: ignore[misc]
     _type: Type[Union[DataSource, Metric, Materialization]],
-    ctx: ParsingContext,
     yaml_dict: Dict[str, Any],
     filename: str,
-    contents: str,
+    contents_by_line: List[str],
 ) -> Any:
     """Parses a model object from (jsonschema-validated) yaml into python object"""
 
     #  Only - add MdoMetadata
```

Comment:
Unrelated, but what is this comment referring to?

Reply:
To be frank, I'm not sure haha. I'll update this to be clearer.